src/public/translations/fr/translation.json

```diff
@@ -111,7 +112,7 @@
     "collapseSubTree": "réduire le sous-arbre",
     "tabShortcuts": "Raccourcis des onglets",
     "newTabNoteLink": "(ou clic central de la souris) sur le lien de la note ouvre la note dans un nouvel onglet",
-    "onlyInDesktop": "Uniquement sur ordinateur (version Electron)",
+    "onlyInDesktop": "Only in desktop (Electron build)",
```

Comment:
@Potjoe-97 , shouldn't this be in French? I see the English translation was carried over instead.